source/Orchestrator/send_notifications.py

```diff
@@ -107,11 +111,7 @@ def lambda_handler(event, _):
         notification.severity = 'ERROR'
         notification.send_to_sns = True
 
-    elif event['Notification']['State'].upper() == 'WRONGSTANDARD':
-        notification = sechub_findings.SHARRNotification('SHARR',AWS_REGION, None)
-        notification.severity = 'ERROR'
-
-    elif event['Notification']['State'].upper() == 'LAMBDAERROR':
+    elif event['Notification']['State'].upper() == {'WRONGSTANDARD' or 'LAMBDAERROR'}:
```

Comment:
This still won't have the same behavior. You're comparing to a set containing one element: `'WRONGSTANDARD'`:
```py
>>> tup = ('WRONGSTANDARD' or 'LAMBDAERROR')
>>> tup
'WRONGSTANDARD'
>>> my_set = {'WRONGSTANDARD' or 'LAMBDAERROR'}
>>> my_set
{'WRONGSTANDARD'}
>>> 'WRONGSTANDARD' == tup
True
>>> 'WRONGSTANDARD' == my_set
False
>>> 'LAMBDAERROR' == tup
False
>>> 'LAMBDAERROR' == my_set
False
>>> other_set = {'WRONGSTANDARD', 'LAMBDAERROR'}
>>> other_set
{'LAMBDAERROR', 'WRONGSTANDARD'}
>>> 'LAMBDAERROR' in other_set
True
>>> 'WRONGSTANDARD' in other_set
True
```